apollo-router/src/plugins/telemetry/config_new/spans.rs

```diff
@@ -556,6 +566,10 @@ mod test {
         assert!(values
             .iter()
             .any(|key_val| key_val.key == opentelemetry::Key::from_static_str("test")));
+
+        assert!(values.iter().any(|key_val| key_val.key
+            == opentelemetry::Key::from_static_str("otel.name")
+            && key_val.value == opentelemetry::Value::String(String::from("new_name").into())));
```

Comment:
Should there also be a test that asserts that when this isn't the case that the default names will be used?

Reply:
There are integration tests that already exist for that